### ISSUE TITLE
Data Source Description Needs To Be Updated

### ISSUE THREAD

**Issue by GoogleCodeExporter** [gzog/analytics-issues#832]
```
Name of related component: Data Source Dimension Description

URL of the documentation page: 
https://developers.google.com/analytics/devguides/reporting/core/dimsmets#view=d
etail&group=platform_or_device&jump=ga_datasource

Issue summary:
The description for the data source dimension needs to be updated. It contains 
the legacy ga.js library which does not set the data source dimension to "web". 
When a hit is received from ga.js the data source dimension gets set to "(not 
set)".

Basically, the ga.js text needs to be removed from the description.

Notes:
The measurement protocol states it correctly. You can find it here: 
https://developers.google.com/analytics/devguides/collection/protocol/v1/paramet
ers#ds
```

Original issue reported on code.google.com by `george.p...@gmail.com` on 22 Dec 2015 at 11:34
